### PR TITLE
Add Rule,Remediation and Test for SLES-15-030780

### PR DESCRIPTION
**Pull request by yarunachalam**
#### Description:

- Add Rule,Remediation and Test for SLES-15-030780

#### Rationale:

- Add SLE15 rule for 'Record Attempts to Alter Process and Session Initiation Information btmp'

### CODE REVIEW

**Comment by jan-cerny** [comment]
@openscap-ci test this please

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_btmp/rule.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+references:
+    stigid@sle15: SLES-15-030780
+    srg: SRG-OS-000472-GPOS-00217
+    disa: CCI-000172
+    nist: AU-12(c),AU-12.1(iv)
+
+ocil_clause: 'Audit rule is not present'
+
+ocil: |-
+    To Check the file is being audited by performing the following command
```
Comment (vojtapolasek) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_btmp/rule.yml:39]:
```suggestion
    Check that the file is being audited by running the following command:
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_btmp/rule.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <pre> sudo auditctl -l | grep -w '/var/log/btmp'</pre>
+template:
```
Comment (vojtapolasek) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_btmp/rule.yml:41]: blank line before template please

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Thank you for the new rule. Just two small things, please see comments.

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_btmp/rule.yml (hunk 2 above)
Comment (yarunachalam) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_btmp/rule.yml:42]: Done

**Pull request merged**